src/workerd/api/actor-state.c++

```diff
@@ -13,6 +13,8 @@
 #include <v8.h>
 #include <workerd/io/actor-cache.h>
 #include <workerd/io/actor-storage.h>
+#include <workerd/api/web-socket.h>
+#include <workerd/io/hibernation-manager.h>
```

Comment:
Welp, just realized we're including the hibernation manager (and using the class) in this commit, but the file + class are introduced in a _later_ commit...

Will fix this at some point today